deploy/dcos-node-datadog-agent.json

```diff
@@ -1,102 +1,102 @@
-            },
-            {
-                "containerPath": "/host/proc",
-                "hostPath": "/proc",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/host/sys/fs/cgroup",
-                "hostPath": "/sys/fs/cgroup",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/conf.d/mesos_slave.d",
-                "hostPath": "/opt/datadog-agent/conf.d/mesos_slave.d",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/etc/cont-init.d/90-mesos.sh",
-                "hostPath": "/opt/datadog-agent/cont-init.d/90-mesos.sh",
-                "mode": "RO"
-            }
-        ],
-        "docker": {
-            "image": "datadog/agent-dev:dcos-auth-token-py3",
-            "network": "BRIDGE",
-            "portMappings": [
-                {
-                    "containerPort": 8125,
-                    "hostPort": 8125,
-                    "servicePort": 10000,
-                    "protocol": "udp",
-                    "labels": {}
-                }
-            ],
-            "privileged": false,
-            "parameters": [
-                {
-                    "key": "env",
-                    "value": "DD_API_KEY=<YOUR_DD_API_KEY>"
-                },
-                {
-                    "key": "env",
-                    "value": "MESOS_SLAVE=true"
-                }
-            ],
-            "forcePullImage": true
-        }
-    },
-    "healthChecks": [
+    "docker": {
+      "image": "datadog/agent-dev:dcos-auth-py3",
+      "network": "BRIDGE",
+      "portMappings": [
         {
-            "protocol": "COMMAND",
-            "command": {
-                "value": "/probe.sh"
-            },
-            "gracePeriodSeconds": 300,
-            "intervalSeconds": 60,
-            "timeoutSeconds": 20,
-            "maxConsecutiveFailures": 3
+          "containerPort": 8125,
+          "hostPort": 8125,
+          "servicePort": 10000,
+          "protocol": "udp",
+          "labels": {}
         }
-    ],
-    "portDefinitions": [
+      ],
+      "privileged": false,
+      "parameters": [
         {
-            "port": 10000,
-            "protocol": "tcp",
-            "name": "default",
-            "labels": {}
+          "key": "env",
+          "value": "DD_API_KEY=<YOUR_DD_API_KEY>"
         },
         {
-            "port": 10001,
-            "protocol": "tcp",
-            "labels": {}
+          "key": "env",
+          "value": "MESOS_SLAVE=true"
         }
-    ]
-}
\ No newline at end of file
+      ],
+      "forcePullImage": true
+    }
+  },
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": {
+        "value": "/probe.sh"
+      },
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 60,
+      "timeoutSeconds": 20,
+      "maxConsecutiveFailures": 3
+    }
+  ],
+  "portDefinitions": [
+    {
+      "port": 10000,
```

Comment:
is that for a mesos service ? I see it referenced [here](https://github.com/DataDog/dcos-datadog-agent-docs/pull/1/files#diff-41c6a2fa5a4ebdf8033e4333b29006c05974cf2814d66e914b7f0631c1e8919dR58)

Reply:
I'll have to confirm where that came from..

Reply:
So the rationale is this http://mesosphere.github.io/marathon/docs/ports.html
But I am not sure if this is correct - in the above section it is targeting the UDP dsd port. @clamoriniere do you know what's up with this (copied from your doc 😄 )